### DISCUSSION TITLE
Selectable Cards

### DISCUSSION THREAD

**Discussion by yoavTaieb** [segmentio/evergreen#1373] [Q&A] [answered]
I cannot find any reference to "selectable cards" or pane in docs or library, but I can find it in the Figma project. How to use it?

From Figma project:

<img width="516" alt="Screen Shot 2021-11-30 at 17 19 52" src="https://user-images.githubusercontent.com/66865717/144074956-747b80f7-71ec-492c-836b-fa895cde25d3.png">

**Comment by kamebkj** [comment, marked as answer]
Hi! Thanks for the question. We currently don't have this component out-of-the-box, but you can create your own by extending the `<Card />` component. The component in the Figma file serves as a best practice on how the design should look like.
